### PR TITLE
Latest ONIX changes from Chris, deprecated captions Now Open/Closed

### PR DESCRIPTION
**Pull request by clapierre**
Here is the rendered view
https://raw.githack.com/w3c/a11y-discov-vocab/Latest-ONIX-Changes-from-Chris/crosswalk/index.html

### CODE REVIEW

File: crosswalk/index.html [hunk 1]
```diff
@@ -856,5 +878,9 @@ <h2>Schema.org and ONIX</h2>
 				</tbody>
 			</table>
 		</section>
+        <section>
+            <h2>Acknowledgements</h2>
+            <p>Christopher Saynor (EDItEUR)</p>
```
Comment (mattgarrish) [inline crosswalk/index.html:883]: If you're only acknowledging one person, you might want to turn this into a paragraph instead. Maybe something like: "The editors would like to thank Christopher Saynor (Editeur) for his invaluable help reviewing this document."

**Comment by madeleinerothberg** [comment]
Some hopefully easy to fix/dismiss questions and issues.

1) Should 
"certifierCredential | Code 93: Compliance certification by"
include "(url)" ?

2) "[Code V213](https://ns.editeur.org/onix/en/175/V213):Sign language" should have a space after the ":" 

3) "[Code 09](https://ns.editeur.org/onix/en/196/09): Inaccessible" is listed alone near the bottom of the accessibilityFeature table in addition to being crosswalked to Schema.org "none" higher up in the same table. Was it on it's own line when it couldn't be crosswalked but now it is better in context or is there a reason to keep the standalone line?

4) Same for Code 22 Language tagging; appears twice in accessibilityFeature table

5) "[Code 10](https://ns.editeur.org/onix/en/196/10): No reading system accessibility options disabled" is listed alone near the bottom of the accessibilityFeature table in addition to being crosswalked elsewhere

6) "[noSoundHazard](https://www.w3.org/2021/a11y-discov-vocab/latest/#noSoundHazard)" row has stray HTML visible after ONIX term

7) "sound risk unknown" should have capital S

8) Was "[Code 42](https://ns.editeur.org/onix/en/81/42): Assessment material" intended to be included in the list of visual access mode equivalents? Is the assessment material assumed to be non-text visuals?

9) In the accessModeSufficient table, the row for textual has stray "ca" at end of ONIX entry

10) I don't understand the structure of the accessibilitySummary table -- is "human-readable text" meant to be a separate row or is it a commentary on accessibilitySummary? As shown, it seems like there are two different fields being crosswalked.

**Comment by chrisONIX** [comment]
In response to questions above:
3- "[Code 09](https://ns.editeur.org/onix/en/196/09): Inaccessible" is listed alone near the bottom of the accessibilityFeature table in addition to being crosswalked to Schema.org "none" higher up in the same table. Was it on its own line when it couldn't be crosswalked but now it is better in context or is there a reason to keep the standalone line?

I would say remove from standalone.

4- Same for Code 22 Language tagging; appears twice in accessibilityFeature table

Agree - remove from standalone 


5 - "[Code 10](https://ns.editeur.org/onix/en/196/10): No reading system accessibility options disabled" is listed alone near the bottom of the accessibilityFeature table in addition to being crosswalked elsewhere
The second instant of code 10 - is code 10 from list 81

**Comment by chrisONIX** [comment]
For accessibilityHazard 
unknown is cross walked to [List: 143; Code: 00](https://ns.editeur.org/onix/en/143/00) No known hazards or warnings
I would suggest that it would be better to map none to [List: 143; Code: 00](https://ns.editeur.org/onix/en/143/00) No known hazards or warnings and leave unknown as a blank - there is no direct equivalent in ONIX - for now in ONIX terms we would recommend using the three precise unknown values in the ONIX record.

**Comment by chrisONIX** [comment]
Possible additions to mapping - 
Tagged PDF - there is now a new code in list 196 for PDF/UA-2 - code 06 - so suggest referencing both code 05 as now and 06 in the cross walk

Not sure if this is an exact equivalent 
timingControl - and code 31 from list 196 - https://ns.editeur.org/onix/en/196/31 
Either cross walk these or add 31 to the end on its own.

**Comment by chrisONIX** [comment]
Three ONIX codes from list 196 to possibly add to end of the accessibility feature list
Code 25 - https://ns.editeur.org/onix/en/196/25 
Use of color is not sole means of conveying information - there is no exact match 

Code 32 - https://ns.editeur.org/onix/en/196/32 
Landmark navigation - I do not think this has an exact match

Code 34 - https://ns.editeur.org/onix/en/196/34
Accessible chemistry content (as MathML)
Code 18 is for ChemML and already cross walked

**Comment by clapierre** [comment]
All of these latest issues found has been addressed including adding the new MathML-latex and MathML-chemistry which was recently added.

**Review by madeleinerothberg — APPROVED** [review]
Looks good!

**Pull request merged**